app/code/Magento/Bundle/Model/Option/SaveAction.php

```diff
@@ -40,22 +44,31 @@ class SaveAction
      */
     private $linkManagement;
 
+    /**
+     * @var StoreManagerInterface
+     */
+    private $storeManager;
+
```

Comment:
Apparently we are not using the `storeManager` in this class, can we remove it?

Reply:
Done